### ISSUE TITLE
AttributeError: can't set attribute

### ISSUE THREAD

**Issue by michaelcaplan** [sphinx-doc/sphinx-intl#4]
Hi,

I'm trying to figure out why i can't update my po files.  I'm not a python guy.  What do you make of this backtrace?

I'm running python 2.7.10

Thanks,

Mike

```
$ sphinx-intl update -d /srv/ddx-docs/cml/_locale -p /srv/ddx-docs/cml/_locale/all -l frNot Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/contextual-help-index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/contextual-help-330-index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/index-cmil330.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/index-cmil400.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-outlook-report/description.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-outlook-report/reading.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-outlook-report/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-outlook-report/parameters.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-outlook-report/open-page.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-report/description.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-report/reading.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-report/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-report/parameters.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/revenue-report/open-page.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/practices/description.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/practices/add-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/practices/list.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/practices/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/practices/starting-balances-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/practices/assigning-taxes-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/practices/field-refs.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/practices/open-preferences-tab.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/practices/open-list-page.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/description.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/default/description.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/default/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/default/files/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/default/file/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/default/partners/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/default/open-tasks/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/default/dashboard/index.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/description.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/transaction/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/credit/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/statements/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/postings/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/estimate/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/payment/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/statement/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/finances/invoice/index-labnext.po
Not Changed: /srv/ddx-docs/cml/_locale/fr/LC_MESSAGES/lab/gnav/description.po
Traceback (most recent call last):
  File "/usr/local/bin/sphinx-intl", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/sphinx_intl/commands.py", line 227, in update
    basic.update(locale_dir, pot_dir, languages)
  File "/usr/local/lib/python2.7/dist-packages/sphinx_intl/basic.py", line 55, in update
    c.update_with_fuzzy(cat, cat_pot)
  File "/usr/local/lib/python2.7/dist-packages/sphinx_intl/catalog.py", line 92, in update_with_fuzzy
    new.fuzzy = True
AttributeError: can't set attribute
```

**Comment by shimizukawa** [comment]
Can you please provide the `pot` and `po` files which cause the error?

**Comment by michaelcaplan** [comment]
Hey @shimizukawa 

I'm guessing that the next set of POT / PO files it is evaluating are the two attached.

Thanks for your help!

Mike

[files.zip](https://github.com/sphinx-doc/sphinx-intl/files/93174/files.zip)

**Comment by shimizukawa** [comment]
Thanks, I'll take a look.

**Comment by shimizukawa** [comment]
Fixed version 0.9.9 has been released. Thanks for your reporting!
